### PR TITLE
update version-vc

### PR DESCRIPTION
**Pull request by Jayzilva**
**Add Verified ID functionalities**

- Change the certificate segment to issue a credential.

- Add a button to the authorization page to navigate to the verifier portal.

### CODE REVIEW

File: src/portals/blockchain-frontend/apps/blockchain-frontend/components/credential-container/CredentialMain.tsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+  const handleSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    try {
+      const response = await submitData(formData);
+      console.log('Form data submitted successfully:', response);
+
+      setPreviewUrl(response.url); // Set preview URL from response
+    } catch (error) {
+      console.error('Error submitting form:', error);
```
Comment (chan4lk) [inline src/portals/blockchain-frontend/apps/blockchain-frontend/components/credential-container/CredentialMain.tsx:17]: Show an error to User, A toast preferably

File: src/portals/blockchain-frontend/apps/blockchain-frontend/components/credential-container/CredentialMain.tsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    }
+  };
+
+  // Server Action (sends data to the provided URL)
+  async function submitData(data: any) {
+    const url =
+      'https://razor-certificate-app.azurewebsites.net/api/issuer/issuance-request';
```
Comment (chan4lk) [inline src/portals/blockchain-frontend/apps/blockchain-frontend/components/credential-container/CredentialMain.tsx:24]: Get endpoints from env variables

**Review by chan4lk — CHANGES_REQUESTED** [review]
Please address the comments

File: src/portals/blockchain-frontend/apps/blockchain-frontend/components/credential-container/CredentialMain.tsx (hunk 1 above)
Comment (Jayzilva) [reply src/portals/blockchain-frontend/apps/blockchain-frontend/components/credential-container/CredentialMain.tsx:19]: Added toast to errors and submissions

**Pull request merged**